crone/src/MixerClient.cpp

```diff
@@ -217,6 +227,7 @@ void MixerClient::SmoothLevelList::setSampleRate(float sr) {
     monitor.setSampleRate(sr);
     cut_aux.setSampleRate(sr);
     ext_aux.setSampleRate(sr);
+    tape_aux.setSampleRate(sr);
```

Comment:
I noticed that none of the ->cut levels have their sample rates set here, is that intentional?

Reply:
Nope! It’s an oversight